frontend/packages/ceph-storage-plugin/src/constants/index.ts

```diff
@@ -9,3 +9,4 @@ export const PODS = 'Pods';
 export const BY_USED = 'By Used Capacity';
 export const BY_REQUESTED = 'By Requested Capacity';
 export const OCS_OPERATOR = 'ocs-operator';
+export const INDEPENDENT_FLAG = 'INDEPENDENT_FLAG';
```

Comment:
Let's try to keep a clear exported API boundary between Console plugins.

1) in `ocs-independent-plugin/src/plugin.ts`

```ts
export const INDEPENDENT_FLAG = 'INDEPENDENT_FLAG';
```

2) add `ocs-independent-plugin/src/index.ts`

```ts
export { INDEPENDENT_FLAG } from './plugin';
```

3) in `ocs-independent-plugin/package.json`

```json
..
"private": true,
"main": "src/index.ts",
"dependencies": ..
..
```

and finally in `ceph-storage-plugin/src/plugin.ts`

```ts
import { INDEPENDENT_FLAG } from '@console/ocs-independent-plugin';
```

Reply:
I did this because. ocs is dependent on ceph already. If I export this from ocs then wouldn't this cause a circular dependency issue?